docs/includes/topic_agent-redundancy-scaleout.rst

```diff
@@ -0,0 +1,93 @@
+:orphan: true
+
+Agent Redundancy and Scale Out
+==============================
+
+Overview
+--------
+
+.. important::
+
+    We refer to 'hosts' a lot in this document. A 'host' could be a Neutron controller, a server, a Docker container, etc.; the important takeaway is that in order to run multiple agents in one environment, **each agent must have a unique** ``hostname``.
+
+When the Neutron LBaaS plugin loads the F5® LBaaSv2 driver, it creates a global messaging queue to be used for all callbacks and status update requests from F5 LBaaSv2 agents. Requests are passed from the global messaging queue to F5 LBaaSv2 drivers in a round-robin fashion, then passed on to an F5 agent as described in the :ref:`Agent-Tenant Affinity` section.
+
+Agent-Tenant Affinity
+`````````````````````
+
+The F5 LBaaSv2 agent scheduler uses the Neutron database and 'agent-tenant affinity' to determine which F5 agent should handle an LBaaS request.
+
+How it works:
+
+#. The Neutron controller receives a new ``loadbalancer`` request.
+#. The F5 LBaaSv2 agent scheduler consults the Neutron database to determine if any F5 agent is already bound to the loadbalancer to which the request applies.
+#. If the scheduler finds a bound agent, it assigns the request to that agent.
+#. If the scheduler doesn't find a bound agent, it checks the loadbalancer's ``tenant_id`` to determine if any agent has already handled a request for that tenant (i.e., has 'agent-tenant affinity').
+#. If the scheduler finds an agent that has affinity with the loadbalancer's tenant, it selects that agent to complete the request.
+#. If the scheduler doesn't find an agent that either is bound to the loadbalancer or has affinity with the loadbalancer's ``tenant_id``, it selects an active agent at random.
+
+    * The selected agent is then bound to the loadbalancer and handles all future LBaaS requests associated with it.
+
+#. If the agent bound to the loadbalancer is inactive, the scheduler looks for other active agents with tenant affinity and assigns the task to the first one it finds. The loadbalancer remains bound to the original agent, with the expectation that the agent will eventually come back online.
```

Comment:
oh good, I needed that clarification. This is different from what the lbaasv1 docs said, but I thought that's what you had explained to us. :)